server/monitor.go

```diff
@@ -352,6 +352,26 @@ func (s *Server) HandleSubsz(w http.ResponseWriter, r *http.Request) {
 	ResponseHandler(w, r, b)
 }
 
+// HandleStacksz processes HTTP requests for getting stacks
+func (s *Server) HandleStacksz(w http.ResponseWriter, r *http.Request) {
+	// Do not get any lock here that would prevent gettign the stacks
+	// if we were to have a deadlock somewhere.
+	var buf []byte
+	size := 10000
```

Comment:
Done. 10,000 seems a lot, but just starting the server without route or client, and we need already more than 4,000. Few clients and we go above 10,000. The question would be: should we limit the size of the buffer? Unfortunately, if we do so, that may render the feature useless if the go routine we are interested in seeing would not be captured. An alternative approach would be to see if it possible to have the runtime dump the stack directly to stderr/logfile instead of requiring a buffer.